src/reanimated2/core.js

```diff
@@ -115,7 +115,7 @@ function workletValueSetter(value) {
     // prevent setting again to the same value
     // and triggering the mappers that treat this value as an input
     // this happens when the animation's target value(stored in animation.current until animation.onStart is called) is set to the same value as a current one(this._value)
-    if (this._value === animation.current) {
+    if (this._value === animation.current && !animation.isHigherOrder) {
```

Comment:
I don't look like a correct approach. I don't think all our animation set their current value to target value.

Reply:
discussed internally